tools/clang/lib/SPIRV/SPIRVEmitter.h

```diff
@@ -390,8 +390,11 @@ class SPIRVEmitter : public ASTConsumer {
   uint32_t translateAPValue(const APValue &value, const QualType targetType);
 
   /// Translates the given frontend APInt into its SPIR-V equivalent for the
-  /// given targetType.
-  uint32_t translateAPInt(const llvm::APInt &intValue, QualType targetType);
+  /// given targetType. If the given APInt value can be represented in 32-bits,
```

Comment:
Need to revert the modifications on `translateAPInt()` declaration and definition?

Reply:
Oops, sorry. Yes.